plugin/forward/README.md

```diff
@@ -88,8 +88,8 @@ forward FROM TO... {
   * `<duration>` - use a different duration for health checking, the default duration is 0.5s.
   * `no_rec` - optional argument that sets the RecursionDesired-flag of the dns-query used in health checking to `false`.
     The flag is default `true`.
-  * `domain DOMAIN` - optional arguments that sets the domain of the dns-query used in health checking.
-    If not configured, the requested domain name is `.`. `DOMAIN` is used to configure the domain name.
+  * `domain DOMAIN` - optional arguments that sets the FQDN of the dns-query used in health checking.
+    If not configured, the requested FQDN is `.`. `DOMAIN` is used to configure the FQDN.
```

Comment:
I think the following reads easier:

```suggestion
  * `domain FQDN` - set the domain name used for health checks to **FQDN**.
    If not configured, the domain name used for health checks is `.`.
```

This should also be updated in the Syntax section...  `health_check DURATION [no_rec] [domain FQDN]`

Reply:
This is better